angular_components/.mono_repo.yml

```diff
@@ -0,0 +1,14 @@
+# See https://github.com/dart-lang/mono_repo for details
```

Comment:
`pub global activate mono_repo` – v1 is out. This file is now called `mono_pkg.yaml`